build.go

```diff
@@ -935,9 +936,12 @@ func NewGometalinter(linter string, dirs ...string) *gometalinter {
 }
 
 func (g *gometalinter) verifyInstalled() {
-	if _, err := runError(g.command, "--disable-all"); err != nil {
-		log.Fatalf("gometalinter is not installed")
+	if !g.verifiedInstallation {
+		if _, err := runError(g.command, "--disable-all"); err != nil {
+			log.Fatalf("gometalinter is not installed")
+		}
 	}
+	g.verifiedInstallation = true
```

Comment:
I don't understand.
How is performing existence checks once before the whole song less beneficial than constructing 3 objects, chaining them multiple times and then calling run just to do nothing?